rest/handler.go

```diff
@@ -283,6 +278,33 @@ func (h *handler) invoke(method handlerMethod, accessPermissions []Permission, r
 		}
 	}
 
+	// Named collections handling
+	if dbc := h.server.GetDbConfig(keyspaceDb); dbc != nil && len(dbc.Scopes) > 0 {
```

Comment:
Why do we need to look up the set of scopes from the dbConfig, as opposed to checking the scopes defined on the dbContext?